spec/absolute.html

```diff
@@ -379,7 +379,7 @@ <h1>Temporal.Absolute.prototype.inTimeZone ( [ _temporalTimeZoneLike_ ] )</h1>
         1. Let _absolute_ be the *this* value.
         1. Perform ? RequireInternalSlot(_absolute_, [[InitializedTemporalAbsolute]]).
         1. If _temporalTimeZoneLike_ is *undefined*, then
-          1. Set _temporalTimeZoneLike_ to *"UTC"*.
```

Comment:
A better way to do this would be to remove the square brackets from around `[ _temporalTimeZoneLike_ ]` in line 373, and then remove the check from lines 382 and 383 altogether. (My understanding is the number of required parameters to a function influences the value of its `.length` property, so that's why line 373 matters.) Same comment for the change in `spec/datetime.html` as well.

Reply:
Makes sense. Updated. Resolving.